### PR TITLE
fix(docker): fix path for the downloaded image

### PR DESCRIPTION
**Pull request by empwilli**

### CODE REVIEW

File: src/docker.rs [hunk 1]
```diff
@@ -42,12 +42,15 @@ pub fn pull_image(name: impl AsRef<str>, arch: Architecture) -> Result<PathBuf>
     let stdout = child.stdout.take().unwrap();
     let mut image_file = File::from(stdout.as_fd().try_clone_to_owned()?);
 
-    let out_path = std::path::PathBuf::from(format!("{}.tar.xz", name.as_ref()));
+    let out_path = std::path::PathBuf::from("./image.tar.xz");
```
Comment (mlilien) [inline src/docker.rs:45]: what if i want to inject 2 different images? is there a move somewhere else in code?

Comment (empwilli) [reply src/docker.rs:45]: Currently, the CLI has no option for multiple images in a single command. Post download/injecting the image file is removed in src/lib.rs:128.

If we want to extend the command to support multiple images in a single go, we probably have to process them either one by one, or use temporary files. I've toyed around with the latter option while development but didn't come to a really satisfactory design yet (temp files being subject to the tempfile cleaner, ...).

Comment (mlilien) [reply src/docker.rs:45]: i don't meant pulling 2 images in one command. 
i mean pulling 2 images consecutively. how does src/lib.rs:128 help?

Comment (empwilli) [reply src/docker.rs:45]: src/lib.rs:128 ensures that these images don't clash. To inject two different images, you can specify their name in the destination path, the cli defaults to "image.tar.gz", if no target name is specified (src/lib.rs:116-118).

**Review by mlilien — APPROVED** [review]

**Review by JanZachmann — APPROVED** [review]

**Pull request merged**